.travis.yml

```diff
@@ -7,14 +7,25 @@ cache:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
     - $HOME/.m2
+env:
+  matrix:
+    - RUN_BARCLAY_TESTS=true
+    - RUN_BARCLAY_TESTS=false
 jdk:
   - oraclejdk8
   - openjdk8
 before_install:
   - wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
   - sudo apt-get -qq update
   - sudo apt-get install -y --no-install-recommends r-base-dev r-recommended qpdf
-script: ./gradlew jacocoTestReport
+script:
+  - if [[ $RUN_BARCLAY_TESTS == true ]]; then
+      echo "Running tests using the Barclay command line parser.";
+      ./gradlew barclayTest;
```

Comment:
Should we also be generating a coverage report for this case? Not sure if barclay and legacy can get out of sync.

Reply:
shouldn't...except for the parser, they are running the same code.